scripts/sonic_sku_create.py

```diff
@@ -0,0 +1,441 @@
+#! /usr/bin/env python2
+"""
+usage: sonic_sku_create.py [-h] [-v] [-f FILE | -m MINIGRAPH] [-b BASE] [-r]
+                           [-c {new_sku_only,l2_mode_only,new_sku_l2}]
+                           [-s SKU] [-p] [-vv]
```

Comment:
>s [](start = 29, length = 1)

Not your fault. Let's make it -k to be consistent with [https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-config-engine/sonic-cfggen](https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-config-engine/sonic-cfggen)